flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -652,19 +654,24 @@ private ModifyOperation convertAlterTableCompact(
     private Operation convertCreateFunction(SqlCreateFunction sqlCreateFunction) {
         UnresolvedIdentifier unresolvedIdentifier =
                 UnresolvedIdentifier.of(sqlCreateFunction.getFunctionIdentifier());
-
+        List<ResourceUri> resourceUris =
+                sqlCreateFunction.getJarPaths().stream()
+                        .map(path -> new ResourceUri(ResourceType.JAR, path))
```

Comment:
It'll always create jar resource. It's fine when we only support jar resource. But as we want to support other resources in the future, here seems is not generic.
Some guys may struggle to modify it when them want to support other resources.